packages/babylon/src/tokenizer/index.js

```diff
@@ -456,6 +462,17 @@ export default class Tokenizer extends LocationParser {
       type = tt.assign;
     }
 
+    if (
+      code === charCodes.asterisk &&
+      next === charCodes.slash &&
+      this.state.hasFlowComment
+    ) {
+      this.state.hasFlowComment = false;
+      this.state.pos += 2;
+      this.nextToken();
+      return;
+    }
```

Comment:
Also this, you can override `readToken_mult_modulo`.

Reply:
Done